flink-scala/src/main/scala/org/apache/flink/api/scala/ExecutionEnvironment.scala

```diff
@@ -110,7 +110,7 @@ class ExecutionEnvironment(javaEnv: JavaEnv) {
     */
   @PublicEvolving
   def getRestartStrategy: RestartStrategyConfiguration = {
-    javaEnv.getRestartStrategy()
+    javaEnv.getRestartStrategy
```

Comment:
Apparently this is not the case in this class. Similar method are also called without parentheses. So lets keep this change to have consistent code in this class.